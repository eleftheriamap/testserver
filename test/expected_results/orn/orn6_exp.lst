
test/exp/test/gen/orn/orn6_exp.bin:     file format binary


Disassembly of section .data:

0000000000000000 <.data>:
   0:	5154339c 	sub	w28, w28, #0x50c, lsl #12
   4:	d15c0d29 	sub	x9, x9, #0x703, lsl #12
   8:	aa29038e 	orn	x14, x28, x9
   c:	8a000000 	and	x0, x0, x0
