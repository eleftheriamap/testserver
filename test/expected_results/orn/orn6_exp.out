Registers:
X00    = 0000000000000000
X01    = 0000000000000000
X02    = 0000000000000000
X03    = 0000000000000000
X04    = 0000000000000000
X05    = 0000000000000000
X06    = 0000000000000000
X07    = 0000000000000000
X08    = 0000000000000000
X09    = ffffffffff8fd000
X10    = 0000000000000000
X11    = 0000000000000000
X12    = 0000000000000000
X13    = 0000000000000000
X14    = 00000000ffff6fff
X15    = 0000000000000000
X16    = 0000000000000000
X17    = 0000000000000000
X18    = 0000000000000000
X19    = 0000000000000000
X20    = 0000000000000000
X21    = 0000000000000000
X22    = 0000000000000000
X23    = 0000000000000000
X24    = 0000000000000000
X25    = 0000000000000000
X26    = 0000000000000000
X27    = 0000000000000000
X28    = 00000000ffaf4000
X29    = 0000000000000000
X30    = 0000000000000000
PC     = 000000000000000c
PSTATE : -Z--
Non-zero memory:
0x00000000 : 0x5154339c
0x00000004 : 0xd15c0d29
0x00000008 : 0xaa29038e
0x0000000c : 0x8a000000
